tools/packaging/common/sidecar.env

```diff
@@ -82,3 +82,21 @@
 # Ignore Istio iptables custom rules
 # Enable this flag if you would like to manage iptables yourself. Default to false (true/false)
 # ISTIO_CUSTOM_IP_TABLES=false
+
+# Location of provisioning certificates. VM provisioning tools must generate a certificate with
+# the expected SAN. Istio-agent will use it to connect to istiod and get fresh certificates.
+PROV_CERT=/var/run/secrets/istio
+
+# Location to save the certificates from the CA. Setting this to the same location with PROV_CERT
+# allows rotation of the secrets. Users may also use longer-lived PROV_CERT, rotated under the control
+# of the provisioning tool.
+# Istiod may return a certificate with additional information and shorter lived, to be used for
+# workload communication. In order to use the certificate with applications not supporting SDS, set this
```

Comment:
What do you mean apps not supporting SDS?  Aren't these all services for VMs?

Reply:
The agent can also be used by 'native Istio' apps - like gRPC - where the app directly uses the certs, without going trough the sidecar. 

In those cases - we can configure the agent to save the certs to files ( will be updated on rotation ). 

I hope the docs are clear that the 'provisioning certs' can be long lived and used only to get shorter-lived certs, which can have additional info ( see the proposal in security to include labels and other info in the certs ).

Reply:
apps not supporting SDS makes sense.  A bit confused why provisioning certs needs to be long lived, isn't it only used at provisioning time?  Do you think reasonable to take a JWT token instead?

Reply:
The problem is that a VM may be down or disconnected for >1hour. 

No, JWT is even shorter lived and long-lived bearer tokens are far more risky. 
In many cases VMs have secure ways to store private keys - TPMs for example, and some
orgs are provisioning the VMs with certificates anyways ( not istio certificates ). 

Between using a long lived token ( like the no-expiration/no-audience 'legacy'/insecure k8s token) and using a long lived cert - it's clearly better to use the cert, even if the VM has no hardware storage ( since we don't have a plugin to take advantage of TPM signing - at this point it's not very relevant, but adding support is not so hard ).